speedcord/shard.py

```diff
@@ -12,6 +12,12 @@
 
 class DefaultShard:
     def __init__(self, shard_id, client, loop: AbstractEventLoop):
+        """
+        Handles all Discord Shard related events. For more information on what sharding is and how it works: https://discord.com/developers/docs/topics/gateway#sharding. 
+        :param shard_id: The id for the shard. Shard ids have the format [<shard_index>, <number_of_shards>], so if you had 3 shards, they would be [0, 3], [1, 3], [2, 3]. 
```

Comment:
shard_id is just the shard_index, not the shard count. Also this is explained in the docs above

Reply:
Aha, gotcha. I'll fix that.

Reply:
Or, rather, just remove the shard count thing.

Reply:
Sounds good